examples/deck/custom-layers/junction-scatterplot-layer.js

```diff
@@ -0,0 +1,45 @@
+// @flow
+import { CompositeLayer, ScatterplotLayer } from 'deck.gl';
+
+export default class JunctionScatterplotLayer extends CompositeLayer {
+  static layerName = 'JunctionScatterplotLayer';
+  static defaultProps = {
+    ...ScatterplotLayer.defaultProps,
+    getFillColor: d => [0, 0, 0, 255],
+    getStrokeColor: d => [255, 255, 255, 255],
+    getInnerRadius: d => 1
```

Comment:
When I think of a stroked points layer, I think of "radius" and "outer radius" as opposed to "inner radius" and "radius". That way it is a strict superset of `ScatterplotLayer` (e.g. `getStrokeColor=null` and `getOuterRadius=null` would produce the same exact results as normal scatterplot layer).